scripts/update-gerbil-nix-recipe.ss

```diff
@@ -15,6 +15,9 @@
 (defonce (default-checkouts-dir)
   (path-maybe-normalize (path-expand "../.." (path-directory (this-source-file)))))
 
```

Comment:
I propose you use the new `(this-source-directory)`

Reply:
Oh, thanks, I'll do that.